backend/src/main/java/hu/flowacademy/meteo/bootstrap/InitDataLoader.java

```diff
@@ -28,10 +28,10 @@
     private final SoilDataRepository soilDataRepository;
     private final WindDataRepository windDataRepository;
     private final StationRepository stationRepository;
-    private final MeasurmentRepository measurmentRepository;
+    private final MeasurementRepository measurementRepository;
 
-    private static final DateFormat DATE_FORMAT_HU = new SimpleDateFormat("yyyy.MM.dd HH:mm", Locale.forLanguageTag("HU-hu"));
-    private static final DateFormat DATE_FORMAT_HU_SPACED = new SimpleDateFormat("yyyy. MM. dd. HH:mm", Locale.forLanguageTag("HU-hu"));
+    private static final DateFormat DATE_FORMAT_HU = new SimpleDateFormat("yyyy.MM.dd HH:mm");
```

Comment:
SimpleDateFormatot ne vegyetek fel konstansnak, mert nem thread safe. A formátum string legyen a konstans ne az object maga. Itt várhatóan sosem fog párhozamosan több threaden futni (persze csak akarni kell, meg lehet oldani), de bármikor másolhatsz innen kódot és akkor már szívás lesz. A formátum konstansokat egyébként ha máshol is használjátok el lehet tenni valami közös helyre. Kár hogy nem láttam meg hamarabb.

Reply:
Itt kérhetek egy pici segítséget hogy pontosan hogy is kellene átírnom?

Reply:
@akesztyus

Reply:
köszönöm.